frontend/src/commponents/semester.jsx

```diff
@@ -1,25 +1,47 @@
-import React, { Component } from "react";
+import React from "react";
 
-class Semester extends Component {
-  renderSuffix() {
-    if (this.props.year === 1) {
-      return "st";
-    } else if (this.props.year === 2) {
-      return "nd";
-    } else if (this.props.year === 3) {
-      return "rd";
+// class Semester extends Component {
+//   renderSuffix() {
+//     if (this.props.year === 1) {
+//       return "st";
+//     } else if (this.props.year === 2) {
+//       return "nd";
+//     } else if (this.props.year === 3) {
+//       return "rd";
+//     } else {
+//       return "th";
+//     }
+//   }
+//   render() {
+//     return (
+//       <button className="btn btn-info cust-btn">
+//         {this.props.year}
+//         {this.renderSuffix()} year
+//       </button>
+//     );
+//   }
+// }
+
+function Semester(props){
+
+  const renderSuffix = () => {
+    if (props.year === 1) {
+      return "st"
+    } else if (props.year === 2) {
+      return "nd"
+    } else if (props.year === 3) {
+      return "rd"
     } else {
-      return "th";
+      return "th"
```

Comment:
don't remove semicolon it helps clarify the code also put proper semicolons where ever possible.